### PR TITLE
Portfolio Page (6.1 in Figma)

### PR DESCRIPTION
**Pull request by CrazyOptimist**
Make portfolio page - 6.1 screen in figma
Note: there might be some mis-matching with figma sheet due to the design changes as known.

Icons are refactored to share same base props, to be displayed in storybook without sub menu in case if icon has no variants, to have export from index file

Changed webpack and tsconfig to write imports without "../../../"

portfolio-detail

[Freedcamp Link](https://freedcamp.com/dashboard/home/panel/task/40026707)

### CODE REVIEW

**Comment by ArthurVogelsang** [comment]
@CrazyOptimist I am not sure any design update after your PR. but need to fix I think
image

**Comment by ArthurVogelsang** [comment]
@CrazyOptimist 
image

File: src/features/nft/portfolio/RarenessScore.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            cy='50%'
+            r={radius}
+            fill='none'
+            stroke='currentColor'
+            strokeWidth={strokeWidth}
+            strokeLinecap='round'
+            strokeDasharray={dashArray}
+            strokeDashoffset={dashOffset}
+          />
+        </svg>
```
Comment (ArthurVogelsang) [inline src/features/nft/portfolio/RarenessScore.tsx:57]: @CrazyOptimist How about you to make ` circle` component?

File: src/features/nft/portfolio/description/Description.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { TNFT } from 'features/nft/Nft.types'
+import { useToggle } from 'react-use'
+import cn from 'classnames'
+import style from 'features/nft/portfolio/description/Description.module.css'
```
Comment (ArthurVogelsang) [inline src/features/nft/portfolio/description/Description.tsx:5]: @CrazyOptimist we can use './Description.module.css' instead of 'feature....'

**Review by ArthurVogelsang — CHANGES_REQUESTED** [review]

File: src/features/nft/portfolio/RarenessScore.tsx (hunk 1 above)
Comment (CrazyOptimist) [reply src/features/nft/portfolio/RarenessScore.tsx:57]: Yes, let me do it.

File: src/features/nft/portfolio/description/Description.tsx (hunk 2 above)
Comment (CrazyOptimist) [reply src/features/nft/portfolio/description/Description.tsx:5]: Yes, it's better :)

File: src/features/nft/portfolio/PortfolioPage.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { TNFT } from 'features/nft/Nft.types'
+import dayjs from 'dayjs'
+import Breadcrumbs from './Breadcrumbs'
+import Header from './Header'
+import Image from './image/Image'
+import Info from 'features/nft/portfolio/info/Info'
+import Description from 'features/nft/portfolio/description/Description'
+import Comments from 'features/nft/portfolio/comments/Comments'
```
Comment (reacteer) [inline src/features/nft/portfolio/PortfolioPage.tsx:9]: Can we use relative paths here and other places?

File: src/features/nft/portfolio/RarenessScore.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import cn from 'classnames'
+
+type TRarenessScoreProps = {
+  title: string
```
Comment (reacteer) [inline src/features/nft/portfolio/RarenessScore.tsx:5]: Let's just use TRarenessScore without Props here and other places.

File: src/features/nft/portfolio/description/Description.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+type TDescriptionProps = {
+  nft: TNFT
```
Comment (reacteer) [inline src/features/nft/portfolio/description/Description.tsx:8]: Let's remove props.

File: src/features/nft/portfolio/comments/Comment.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { TComment } from 'features/nft/portfolio/comments/Comment.types'
+
+type TCommentProps = {
+  comment: TComment
```
Comment (reacteer) [inline src/features/nft/portfolio/comments/Comment.tsx:5]: Let's remove props.

File: src/features/nft/portfolio/image/Image.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import style from 'features/nft/portfolio/image/Image.module.css'
+import { BoxArrowUp, HeartFilled } from 'common/components/Icons'
+
+type TImageProps = {
+  image: string
```
Comment (reacteer) [inline src/features/nft/portfolio/image/Image.tsx:6]: Let's remove props.

File: src/features/nft/portfolio/info/Button.tsx [hunk 8]
```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import cn from 'classnames'
+
+type TButtonProps = {
+  className?: string
```
Comment (reacteer) [inline src/features/nft/portfolio/info/Button.tsx:5]: Let's remove props.

File: src/features/nft/portfolio/info/Info.tsx [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+import { TNFT } from 'features/nft/Nft.types'
+import { HeartFilled, Eye } from 'common/components/Icons'
+import dayjs from 'dayjs'
+import { formatDatesDistance } from 'common/utils/format'
+import Button from './Button'
+import Row from './Row'
+import { useToggle } from 'react-use'
+
+type TInfoProps = {
+  nft: TNFT
```
Comment (reacteer) [inline src/features/nft/portfolio/info/Info.tsx:11]: Let's remove props.

**Review by reacteer — CHANGES_REQUESTED** [review]

File: src/features/nft/portfolio/RarenessScore.tsx (hunk 1 above)
Comment (CrazyOptimist) [reply src/features/nft/portfolio/RarenessScore.tsx]: fixed

File: src/features/nft/portfolio/PortfolioPage.tsx (hunk 3 above)
Comment (CrazyOptimist) [reply src/features/nft/portfolio/PortfolioPage.tsx]: fixed!

File: src/features/nft/portfolio/description/Description.tsx (hunk 2 above)
Comment (CrazyOptimist) [reply src/features/nft/portfolio/description/Description.tsx]: fixed

**Review by reacteer — APPROVED** [review]

**Review by ArthurVogelsang — APPROVED** [review]

**Pull request merged**